src/router/handlers/utils.ts

```diff
@@ -5,5 +5,31 @@
  */
 
 import { createHash } from 'crypto';
+import * as xss from 'xss';
 
-export const hash = (o: any): string => createHash('sha256').update(JSON.stringify(o)).digest('hex');
+const options = {
+    whiteList: {
```

Comment:
Nit-pick: non-inclusive text. Although it's hard to change it since it's an attribute from a dependency.